include/nbl/video/utilities/IPropertyPool.h

```diff
@@ -21,8 +26,7 @@ class NBL_API2 IPropertyPool : public core::IReferenceCounted
 	public:
 		using PropertyAddressAllocator = core::PoolAddressAllocatorST<uint32_t>;
 
-        static inline constexpr auto invalid = PropertyAddressAllocator::invalid_address;
-
+        static inline constexpr uint64_t invalid = 0; 
```

Comment:
when did we agree to change the `invalid` from the AddressAllocator invalid (0xdeadbeefu) to 0?